vars/neoDeploy.groovy

```diff
@@ -59,16 +59,36 @@ import static com.sap.piper.Prerequisites.checkScript
      * @mandatory for deployMode=warParams
      */
     'runtimeVersion',
-        /**
+    /**
      * Compute unit (VM) size. Acceptable values: lite, pro, prem, prem-plus.
      * @parentConfigKey neo
      */
     'size',
-        /**
+    /**
      * String of VM arguments passed to the JVM.
      * @parentConfigKey neo
      */
-    'vmArguments'
+    'vmArguments',
+    /**
+     * Boolean to enable/disable invalidating the cache after deployment.
+     * @parentConfigKey neo
+     */
+    'invalidateCache',
+    /**
+     * Portal landscape region subscribed to in SAP Cloud Platform.
+     * @parentConfigKey neo
+     */
+    'portalLandscape',
+    /**
+     * UsernamePassword type credential containing SAP CP OAuth client id and client secret.
+     * @parentConfigKey neo
+     */
+    'oauthCredentialId',
+    /**
+     * Site ID of the SAP Fiori Launchpad containing the SAP Fiori app. If not set, the cache of the default site, as defined in the Portal service, is invalidated.
```

Comment:
That is only in the comment.

Reply:
This comment goes into the documentation. The docu is generated from this.

Reply:
you mean for client ID?

Reply:
I mean the comments on top of each parameter. They are used when generating the documentation. Because you write that the discrepancy is only in the comment. My point is that this will be in the documentation later. Did I misunderstand something?

Reply:
OK, could you check now?

Reply:
Basically,  I don't understand, what is your point exactly referring to here? You mentioned `ID ` is capitalized for `siteId`. Yes, it is in comment. So, I am also capitalizing `ID `when I call `client ID` in the comment again. But, I don't understand why I should capitalize ID in `oauthCredentialId`? I hope it is fine now. Please have a look again.

Reply:
You are right, I expressed myself not clearly enough! You understood it correctly finally. I did mean the parameter description of `oauthCredentialId`, not that you should change any variable names. Sorry for the confusion.